bot.js

```diff
@@ -741,8 +741,25 @@ function command(message) {
         );
         log.info("Help is for command: " + target);
         if (Object.keys(config.helpTopics).includes(target)) {
-            const detailsObject = config.helpTopics[target];
+            let detailsObject = config.helpTopics[target];
             let response;
+            if (detailsObject.alias != null) {
```

Comment:
Question - Should this be 'while'?

I missed this when aliasing custom commands (an alias cannot point at an alias), and decided against it for technical reasons in core commands (as it would require a linear search over all aliases) and for philosophical reasons in the same (one alias change should not have the potential to butterfly-effect into changing core functionality for many trigger phrases)...

But would it make sense to allow help topic aliases to themselves refer to aliases?

Reply:
I don't see any reason why not, I'll update.